src/projectscene/internal/projectviewstate.cpp

```diff
@@ -558,6 +831,37 @@ int ProjectViewState::trackDefaultHeight() const
     return TRACK_DEFAULT_HEIGHT;
 }
 
+int ProjectViewState::calculateVerticalRulerWidth() const
+{
+    constexpr int MIN_RULER_WIDTH = 56;
+
+    const project::IAudacityProjectPtr prj = globalContext()->currentProject();
+    const IProjectViewStatePtr viewState = prj ? prj->viewState() : nullptr;
```

Comment:
Something strange. You're already in the viewState class, but you're getting viewState from the project. Looks like a copy-paste issue